src/components/desktop/DesktopLayout.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components'
+
+import MapPage from '../map/MapPage'
+import Header from './Header'
+
+const DesktopContainer = styled.div`
+  height: 100%;
+`
+
+const MapContainer = styled.div`
+  height: 100%;
+`
```

Comment:
What do you think of writing a utility `Container` component here since we're using the same attributes to style both `DesktopContainer` and `MapContainer`?

Reply:
Things do get changed here: https://github.com/hack4impact-uiuc/falling-fruit/pull/20/files

The styling in this PR is actually a bit wrong because there's a scrollbar in the desktop layout (the map is too tall). But that gets fixed with flexbox in the pane PR.